src/component-queries/convert-data.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    DiseaseCellLineNode,
+    NormalCellLineNode,
+    UnpackedDiseaseCellLine,
+    UnpackedNormalCellLine,
+} from "./types";
+
```

Comment:
before there was a mixture of snake case that came from the database, and new properties that were camel cased. Here i'm just fully unpacking all the data from the query into a flat object